vision/spec/detect_landmarks_spec.rb

```diff
@@ -6,39 +6,48 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-# Unless required by applicable law or agreed to in writing, software
+# Unless required by applicable law or agreed to in write, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-require_relative "../detect_landmarks"
 require "rspec"
-require "tempfile"
+require "google/cloud/storage"
+
+require_relative "../detect_landmarks"
 
-describe "Detect Landmarks Sample" do
+describe "Detect Landmarks" do
+
+  before do
+    @storage    = Google::Cloud::Storage.new
```

Comment:
This should use project: ENV["GOOGLE_CLOUD_PROJECT"].. if you're going to use @project_id = ENV["GOOGLE_CLOUD_PROJECT"].